resources/custom-resources/disable-instance-protection/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/DisableInstanceProtection.java

```diff
@@ -55,8 +55,8 @@ public Object handleRequest(Map<String, Object> event, Context context) {
         LOGGER.info("Processing for Autoscaling group {}", autoScalingGroup);
         try {
             Runnable r = () -> {
-                if ("Delete".equalsIgnoreCase(requestType)) {
-                    LOGGER.info("DELETE");
+                if ("Delete".equalsIgnoreCase(requestType) || "Update".equalsIgnoreCase(requestType)) {
+                    LOGGER.info("Request Type: " + requestType);
```

Comment:
These changes don't appear to be related to fixing billing. Can you explain why this and the change to saas-boost-svc-onboarding.yaml are in the diff?

Reply:
The current flow is when a tenant is updated in UI, it triggers a CloudFormation Stack Update. This change is necessary to send the message to the BillingService with the Billing Plan in case the Billing Plan has changed. The Billing Service will determine if the plan has changed.